src/main/resources/yamlschema/BatchResponse.yaml

```diff
@@ -0,0 +1,25 @@
+#version: 0.0.3
```

Comment:
All of these "new" files are split out from `PrintProviderSchema.yaml` above, which was initially added in PR https://github.com/cabinetoffice/eip-ero-print-api/pull/1